build/configs/rtl8721csm/tc/defconfig

```diff
@@ -387,6 +387,11 @@ CONFIG_USERMAIN_STACKSIZE=2048
 CONFIG_PTHREAD_STACK_MIN=256
 CONFIG_PTHREAD_STACK_DEFAULT=2048
 
+#
+#SRAM Heap size information
+#
+CONFIG_SRAM_HEAP_SIZE=38
```

Comment:
The available heap for this config before patch is "40192". So the "CONFIG_SRAM_HEAP_SIZE" value can only be maximum of (available heap/1024) or else we will get a overflow compile error. Then may need to adjust accordingly.